Traceback (most recent call last):
  File "/home/va/Documents/remote/google-maps-scraper/venv/lib/python3.11/site-packages/bose/base_task.py", line 117, in run_task
    self.run(driver)
  File "/home/va/Documents/remote/google-maps-scraper/src/scraper.py", line 229, in run
    result = get_data()
             ^^^^^^^^^^
  File "/home/va/Documents/remote/google-maps-scraper/src/scraper.py", line 208, in get_data
    links = get_links(q)
            ^^^^^^^^^^^^
  File "/home/va/Documents/remote/google-maps-scraper/src/scraper.py", line 126, in get_links
    links = extract_links(els)
            ^^^^^^^^^^^^^^^^^^
  File "/home/va/Documents/remote/google-maps-scraper/src/scraper.py", line 123, in extract_links
    return list(map(extract_link, elements))
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: 'NoneType' object is not iterable
